classes/jsp/hugo_repo.jsp

```diff
@@ -24,9 +24,9 @@
                 status = "pulled ok";
             }
 
-			String parameters = " -D --destination " + Gitea.self.getHome() + "/custom/public/www" + repo;
+			String parameters = " -D --destination " + Gitea.self.getHome().resolve("custom").resolve("public").resolve("www"+repo);
```

Comment:
I worry about this as can `repo` contain a slash? I defer to @deleolajide for merge approval.

Reply:
I'm not sure about this either. I _assume_ that there is no slash between `www` and repo, but Dele can probably tell us.